compiler/compiler.py

```diff
@@ -343,7 +350,9 @@ def _store_compilation_result(status: Task, out_path: Optional[str],
 # TODO: use ``docker`` Python API instead of subprocess.
 # TODO: rename []_dvips_flag parameters when we figure out what they mean.
 # TODO: can we get rid of any of these?
-def _run(source: SourcePackage, output_format: Format = Format.PDF,
+def _run(source: SourcePackage,
+         stamp_label: str, stamp_link: str,
```

Comment:
```suggestion
         stamp_label: Optional[str], stamp_link: Optional[str],
```

Since the caller may not be adding these.

Reply:
Done